feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -1,51 +1,394 @@
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray600
+                    )
+                    Spacer(modifier = Modifier.width(4.dp))
+                    Text(
+                        text = data.jurisdiction,
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray800
+                    )
+                }
+            }
+        }
     }
 }
 
+@Composable
+fun CalendarComponent(
+    currentMonth: YearMonth,
+    selectedDate: LocalDate,
+    onMonthChange: (YearMonth) -> Unit,
+    onDateSelected: (LocalDate) -> Unit,
+) {
+    val days = remember(currentMonth) { generateDaysForMonth(currentMonth) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        // 상단: 년/월 & < > 버튼
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 5.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = stringResource(
+                    R.string.calender_component_date,
+                    currentMonth.year.toString(),
+                    currentMonth.monthValue.toString()
+                ),
+                style = CrowdZeroTheme.typography.h3Bold,
+                color = CrowdZeroTheme.colors.green700
+            )
+
+            Spacer(modifier = Modifier.width(225.dp))
+
+            Icon(
+                painter = painterResource(id = R.drawable.calender_button_l),
+                tint = CrowdZeroTheme.colors.green700,
+                contentDescription = "Next Month",
+                modifier = Modifier
+                    .size(16.dp)
+                    .clickable { onMonthChange(currentMonth.minusMonths(1)) }
+            )
+
+            Spacer(modifier = Modifier.width(15.dp))
+
+            Icon(
+                painter = painterResource(id = R.drawable.calender_button_r),
+                tint = CrowdZeroTheme.colors.green700,
+                contentDescription = "Next Month",
+                modifier = Modifier
+                    .size(16.dp)
+                    .clickable { onMonthChange(currentMonth.plusMonths(1)) }
+            )
+        }
+
+        Spacer(modifier = Modifier.height(6.dp))
+
+        Spacer(
+            modifier = Modifier
+                .height(1.dp)
+                .fillMaxWidth()
+                .background(CrowdZeroTheme.colors.gray500)
+        )
+
+        Spacer(modifier = Modifier.height(15.dp))
+
+        // 요일 헤더
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceAround
+        ) {
+            listOf("일", "월", "화", "수", "목", "금", "토").forEach { day ->
```

Comment:
P1: 변하지 않는 객체이기 때문에 immutableListOf로 수정해주세요!! 그래야 성능이 좋아집니다
```suggestion
            immutableListOf("일", "월", "화", "수", "목", "금", "토").forEach { day ->
```